github-sample/workflows/build-publish.yml

```diff
@@ -1,5 +1,5 @@
 # File generated by gflows, do not modify
-# Source: .gflows/workflows/build-publish
+# Source: .gflows\workflows\build-publish
```

Comment:
I think this should be unchanged. Using gflows in Windows updates the path like this unfortunately.